### DISCUSSION TITLE
Source repository for Linux Mint's modified packages?

### DISCUSSION THREAD

**Discussion by matejcik** [linuxmint/discussions#51] [Development]
Where do build sources for things like `chromium` live?

I would love to send a PR to add a certain build flag (my [issue](https://github.com/linuxmint/linuxmint/issues/527) is completely ignored for several months) but can't find where. Searching the list of LinuxMint repositories has proven futile so far.

**Comment by mtwebster** [comment]
Our repos for browser builds are currently private (as they include build infrastructure also that can't be public). But all files specific to the build are included in the downloadable source -
http://packages.linuxmint.com/search.php?release=victoria&section=any&keyword=chromium

I've responded to your original issue as well.
